uavcan/time/62804.Synchronization.1.0.uavcan

```diff
@@ -0,0 +1,21 @@
+#
+# Network-wide time synchronization message.
+# Any node that publishes timestamped data should use this time reference.
+#
+# This message should always be published at the highest priority level.
```

Comment:
Perhaps we should reserve 16 Ports with the lowst ID and use one of them for this message?

Reply:
I am not a big fan of splitting the ranges like that. The time synchronization protocol doesn't really require the highest priority, the comment is misleading and I added it a long time ago by mistake. Let me remove it now. The background is available at https://www.can-cia.org/fileadmin/resources/documents/proceedings/1994_gergeleit.pdf